numpy/__init__.py

```diff
@@ -182,9 +182,36 @@
     oldnumeric = 'removed'
     numarray = 'removed'
 
-    # We don't actually use this ourselves anymore, but I'm not 100% sure that
-    # no-one else in the world is using it (though I hope not)
-    from .testing import Tester
+    if sys.version_info[:2] >= (3, 7):
```

Comment:
Worth a comment like `# module level getattr is only supported in 3.7 onwards`

Reply:
done.